Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+@Library('jenkins.shared.library') _
+
+environment {
+  HELM_IMAGE = "infoblox/helm:3.2.4-5b243a2"
+  REGISTRY = "core-harbor-prod.sdp.infoblox.com"
+  TAG = sh(script: "git describe --always --long --tags", returnStdout: true).trim()
+}
+
+pipeline {
+  agent {
+    label 'ubuntu_docker_label'
+  }
+  stages {
+    stage("Build Image") {
+      sh "docker build . -t tugger:${version}"
+    }
+    stage("Push Image") {
+      when {
+        anyOf {
+          branch 'master'
+          branch 'jenkinsfile'
```

Comment:
I assume this `jenkinsfile` branch is just here for testing, but once the build has been validated, it should just be `branch 'master'`

Reply:
We use this pattern in a lot of repos. By leaving `jenkinsfile` in there, it allows testing changes to the Jenkinsfile as long as you do it from the `jenkinsfile` branch. It's useful every time we need to modify the Jenkinsfile, not just the first PR.